apps/frontend-manage/src/components/analytics/performance/TotalStudentPerformancePlot.tsx

```
@@ -0,0 +1,195 @@
+          }))}
+          csvFilename={`${courseName.replace(' ', '-')}_participant_performance`}
+          className={{
+            tableHeader: 'h-7 p-2',
+            tableCell: 'h-7 p-2',
+          }}
+        />
+      </div>
+    </div>
+  )
+
+  return null
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)